README.rst

```diff
@@ -51,6 +51,11 @@ And another for writing one:
             with pyorc.Writer(data, "struct<col0:int,col1:string>") as writer:
                 writer.write((1, "ORC from Python"))
 
+Reading from commandline:
+
+`python3 -c "import pyorc; import sys; print(list(pyorc.Reader(open(sys.argv[1],'rb'))))"  file.orc`
```

Comment:
It will load the entire file into memory...
A simple CLI could be added to the library so that we don't also see apache-tools.